containers/Editor.js

```diff
@@ -0,0 +1,168 @@
+// Theirs
+import { Container } from 'unstated'
+import domtoimage from 'dom-to-image'
+
+// Ours
+import api from '../lib/api'
+import {
+  DEFAULT_EXPORT_SIZE,
+  EXPORT_SIZES_HASH,
+  DEFAULT_CODE,
+  DEFAULT_SETTINGS
+} from '../lib/constants'
+import { serializeState } from '../lib/routing'
+import { getState, isImage } from '../lib/util'
+
+class EditorContainer extends Container {
+  constructor(props = {}) {
+    super(props)
+    this.state = {
+      ...DEFAULT_SETTINGS,
+      uploading: false,
+      code: props.content
+    }
+
+    this.save = this.save.bind(this)
```

Comment:
are any of these private? If so, maybe specify which are

Reply:
None of them are as of now. Just rename one last one